HeliumExample/src/main/res/values/strings.xml

```diff
@@ -6,6 +6,7 @@
     <!-- Helium App Placement -->
     <string name="heliumInterstitial">CBInterstitial</string>
     <string name="heliumRewarded">CBRewarded</string>
+    <string name="heliumBanner">CBBanner2</string>
```

Comment:
Is `CBBanner2` right?

Reply:
This placement is setup for Fyber, but anyone should work, also tested with CBBanner